lib/replset.js

```diff
@@ -368,22 +366,19 @@ class ReplSet extends EventEmitter {
 
     return new Promise(function(resolve, reject) {
       co(function*() {
```

Comment:
FWIW, these are also cases where we no longer have to wrap this in a `Promise`, but just return the `co` call